src/platform/i386/vga.c

```diff
@@ -161,15 +161,19 @@ cls(void)
 }
 
 /*
- * Clear the screen and initialize VIDEO, XPOS and YPOS.
  * VIDEO virtual address set to HIGH address.
  */
 void
-vga_init(void)
+vga_high_init(void)
 {
-	int i = 0;
-
 	video = chal_pa2va(VIDEO);
+}
+
+/* Clear the screen and initialize VIDEO, XPOS and YPOS. */
+void
+vga_init(void)
+{
+	video = (unsigned char *) VIDEO;
```

Comment:
This is great.  We can now also map this into the booter, and it can pass it on to another component.  Would be nice to move all of the print infrastructure out of the kernel.  Thanks for enabling this!